### ISSUE TITLE
popover dynamic content not clean when do popover('show') again

### ISSUE THREAD

**Issue by Awakening-j** [twbs/bootstrap#19656]
popover dynamic content not clean when do popover('show') again,  
and the content will add together, so click n times, popover content will have n element  
![2016-04-01 15 52 23](https://cloud.githubusercontent.com/assets/8085088/14201023/ca649ff8-f821-11e5-8630-0ea30d0d627d.png)
![2016-04-01 15 53 27](https://cloud.githubusercontent.com/assets/8085088/14201053/eea2dcf4-f821-11e5-95ab-9aa8e38e5fc6.png)

**Comment by cvrebert** [comment]
@Awakening-j Please post a JS Fiddle or JS Bin that demonstrates the problem.
(Also, protip: Post code as text rather than a screenshot of text. I can't copy-paste code from a screenshot.)

**Comment by Awakening-j** [comment]
ok, i will post a demo later.  
Have you met this problem before?   
I'm using bootstrap's popover in `angular directive`, (for some reason, i'm not using angular-ui popover)

**Comment by Awakening-j** [comment]
@cvrebert I'm using bootstrap 3.1.1, how can i find the cdn source code link

**Comment by Awakening-j** [comment]
@cvrebert here is the [demo](http://plnkr.co/edit/CmdcnDm2sZFh3SycKZV6?p=preview)

**Comment by twbs-lmvtfy** [comment]
Hi @Awakening-j!

You appear to have posted a live example (http://run.plnkr.co/plunks/CmdcnDm2sZFh3SycKZV6/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W001](https://github.com/twbs/bootlint/wiki/W001): `<head>` is missing UTF-8 charset `<meta>` tag
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness
* line 14, column 5: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 6, column 5: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.1.1
* line 7, column 5: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.1.1

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by Awakening-j** [comment]
@twbs-lmvtfy hei, are you robot ?  
1. I've added utf-8  
2. i'm using chrome
3. no need responsiveness
4. i've added type="button"
5. I'm using v3.1.1, so I think test with this version will be better

@cvrebert

**Comment by Awakening-j** [comment]
@cvrebert another [demo](http://plnkr.co/edit/4HlLy6IiSF68d6EiAYeW?p=preview) using latest version of bootstrap

**Comment by twbs-lmvtfy** [comment]
Hi @Awakening-j!

You appear to have posted a live example (http://run.plnkr.co/plunks/4HlLy6IiSF68d6EiAYeW/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
A `content` function must return a string, not a number.
Seems to work fine once you fix that: http://plnkr.co/edit/xIvKoctW7YeBpor3T1SD?p=preview
Duplicate of #15400.

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://run.plnkr.co/plunks/xIvKoctW7YeBpor3T1SD/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)
